DMTR-53.tex

```diff
@@ -147,47 +147,297 @@ \subsection{Summary Table}
+reported from the generated catalogs.
+
+
+\textit{Each DIASource record contains required attributes: FAILED}
+
+DMS-REQ-0269 requires many attributes, which are summarized in Table
+\ref{tab:diasrc}.  We determined which attributes were present by manually
+examining a randomly selected DIASource record retrieved from the Butler.
+
+\begin{table}[h]
+	\begin{tabular}{|p{0.4\textwidth}|c|p{0.4\textwidth}|}
+		\hline
+		Attribute & Pass/Fail & Comment \\
+		\hline\hline
+the identity of the Difference Exposure from which it was derived & FAIL & Only implicitly available by reconstructing dataId \\
+		\hline
+	the identity of the associated SSObject, if any & FAIL & Out of scope for this test; but no column is present \\
+		\hline
+		the identity of the parent Source from which this DIASource has been deblended, if any & PASS & \texttt{parent} column is present \\
+		\hline
+		epoch of the observation & FAIL & not present in the catalog (requires denormalization from higher-level metadata) \\
+		\hline
+	focal plane position centroid and error (pixel) & PASS &
+		\texttt{base\_SdssCentroid\_x},
+		\texttt{base\_SdssCentroid\_y},
+		\texttt{base\_SdssCentroid\_xSigma}, \&
+		\texttt{base\_SdssCentroid\_ySigma} \\
+		\hline
+		sky position and associated error (radec) & FAIL & \texttt{coord\_ra} and \texttt{coord\_dec} are present, but no error is reported \\
+		\hline
+	SNR of the detection & FAIL & no SNR is reported \\
+		\hline
+		calibrated PS flux and associated error & PASS & \texttt{base\_PsfFlux\_flux} and \texttt{base\_PsfFlux\_fluxSigma} \\
+		\hline
+	likelihood of the observed data given the PS model & False & not present in the catalog \\
+		\hline
+		calibrated aperture flux and associated error & PASS & several values of \texttt{base\_CircularApertureFlux\_*} are present with errors \\
+		\hline
+	calibrated flux and associated error for a trailed source model & FAIL & not present in the catalog \\
+		\hline
+	length and angle of the trail & FAIL & not present in the catalog \\
+		\hline
+		flux and associated parameters for a dipole model & PASS & several dipole flux values are present \\
+		\hline
+		parameters of an adaptive shape measurement and associated error & PASS & \texttt{base\_SdssShape\_*} \\
+		\hline
+	a measure of source extendedness & FAIL & not present in the catalog \\
+		\hline
+	the estimated background at the position of the object in the template image with associated uncertainty & FAIL & not present in the catalog \\
+		\hline
+	a measure of spuriousness & FAIL &  not present in the catalog \\
+		\hline
+	flags indicating problems encountered while computing the aforementioned attributes & PASS & many flags are present \\
+		\hline
+  calibrated flux and associated error for the DIAsource as measured on the Visit image & FAIL & not present in the catalog \\
+		\hline
+\end{tabular}
+	\caption{Required DIASource attributes for AG-00-20 \label{tab:diasrc}}
+\end{table}
```

Comment:
I'm actually not sure how we should handle the above: I do think it's an appropriate test, since we need to be sure that we're going to satisfy the requirement, but since it bundles together a large number of measurements and other data I don't think we can easily lump it all under a single JIRA ticket. I'm inclined to suggest to the CCB that they regard this as a pass on the basis that (for this stage in development) these failures are "noncritical", but I wonder if @womullan would prefer a different approach.

Reply:
I'm open to suggestions.